msg/tools/generate_microRTPS_bridge.py

```diff
@@ -77,18 +85,26 @@ def get_absolute_path(arg_parse_dir):
 args = parser.parse_args()
 msg_folder = get_absolute_path(args.msgdir)
 msg_files_send = []
+msg_files_receive = []
 if args.send:
     msg_files_send = [get_absolute_path(msg) for msg in args.send]
-else:
-    msg_files_send = []
 if args.receive:
     msg_files_receive = [get_absolute_path(msg) for msg in args.receive]
+
+if args.package[0] != px_generate_uorb_topic_files.PACKAGE:
+    package = args.package[0]
 else:
-    msg_files_receive = []
+    package = px_generate_uorb_topic_files.PACKAGE
+
 agent = args.agent
 client = args.client
+# If not specified, auto generate the idl filename_send_msgs
+if args.idl is None or args.idl == "":
```

Comment:
If you add a `default=False` to `parser.add_argument("-i", "--no-idl"`, you can directly set `idl = args.idl`

Reply:
true. thanks